Mathlib/Algebra/GroupWithZero/Defs.lean

```diff
@@ -159,17 +159,17 @@ class MulDivCancelClass (M₀ : Type*) [MonoidWithZero M₀] [Div M₀] : Prop w
   protected mul_div_cancel (a b : M₀) : b ≠ 0 → a * b / b = a
 
 section MulDivCancelClass
-variable [MonoidWithZero M₀] [Div M₀] [MulDivCancelClass M₀] {a b : M₀}
+variable [MonoidWithZero M₀] [Div M₀] [MulDivCancelClass M₀]
 
-@[simp] lemma mul_div_cancel_right₀ (a : M₀) (hb : b ≠ 0) : a * b / b = a :=
+@[simp] lemma mul_div_cancel_right₀ (a : M₀) {b : M₀} (hb : b ≠ 0) : a * b / b = a :=
```

Comment:
If you want to stick to the existing order:
```suggestion
@[simp] lemma mul_div_cancel_right₀ {b : M₀} (a : M₀) (hb : b ≠ 0) : a * b / b = a :=
```

Reply:
You are right: I get confused so easily with `variable`!  My reasoning for not wanting to change the order was mostly since I expected potential breaking further downstream.  Here nothing broke...  😊 

I'll leave it up to you, here and in the other place: choose the order that you prefer and it seems like it won't affect much.

Reply:
Let's leave this in alphabetical order, and change the other as well.